integration/js/utils/SdkBaseTest.js

```diff
@@ -20,6 +20,10 @@ class SdkBaseTest extends KiteBaseTest {
       this.userArn = this.payload.userArn;
     }
 
+    if (testName === 'MediaCapture') {
+      this.region = this.payload.region;
```

Comment:
So, currently, we set the region directly through URL for some tests and other stages, so is there a way or should we set the `this.region` value to whatever is the selected one instead of empty when the test is not a 'MediaCapture' test?

Reply:
We are using the [region in the test](https://github.com/aws/amazon-chime-sdk-js/pull/1606/files#diff-bf5edd9d104e0847b257ff2fa050a7d5921edb0c5675f80fc9726ba9afbc59cbR28) to select the region as a step instead of using it directly from the URL before starting the meeting/adding the attendee. We are doing this because the `MediaCapture` feature is available in more than 18 regions and it's not required to maintain a separate URL for each region since we have the region param available in the test config.

This won't effect the other tests as it only does it for `MediaCapture` test and other test don't have the step to set the region. They just auto-populate the region from the URL and don't change it in the actual test.

Reply:
Yup, thanks for the explanation, this wont affect other test but since we are adding `region` field, checking if we can keep it updated with whatever region is selected or if you custom select a different one so that we have a valid region populated for all tests. @ltrung could you pleas help with any suggestions? I have no strong opinion but want to check once.

Reply:
Synced with you offline.